src/protocol/http/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+//!
```

Comment:
Are you planning to add some docs here before mereg?

Reply:
Yeah - forgot about that. Will add some.